pkg/core/native/oracle.go

```diff
@@ -119,7 +119,7 @@ func newOracle() *Oracle {
 	o.AddMethod(md, desc)
 
 	desc = newDescriptor("verify", smartcontract.BoolType)
-	md = newMethodAndPrice(o.verify, 100_0000, smartcontract.NoneFlag)
+	md = newMethodAndPrice(o.verify, 100_0000, smartcontract.All)
```

Comment:
But why? It doesn't notify, doesn't call, doesn't write states, doesn't read states.

Reply:
That was done by mistake. Fixed.